pkg/verify/verify.go

```diff
@@ -256,7 +261,11 @@ func (v *releaseVerifier) Verify(ctx context.Context, releaseDigest string) erro
 		case "http", "https":
 			copied := *store
 			copied.Path = path.Join(store.Path, name)
-			if err := checkHTTPSignatures(ctx, v.client, copied, maxSignatureSearch, verifier); err != nil {
+			client, err := v.clientBuilder.HTTPClient()
+			if err != nil {
+				return err
+			}
```

Comment:
I think we are createing an http client for each store, maybe we can do it once per Verify call ?

Reply:
Absolutely. I moved client creation out of the for loop.